src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
```

Comment:
u don't need to create separate file for each type, btw `react-app-env.d.ts` u have this file for types, and u don't need to import anything from it